moveit_configs_utils/config/trajopt_planning.yaml

```diff
@@ -0,0 +1,8 @@
+planning_plugin: trajopt_interface/TrajOptPlanner
```

Comment:
I'm not sure what the state of traj_opt is but I don't think it's used enough to justify being here.